static/css/style.css

```diff
@@ -98,10 +103,12 @@ a:hover {
     right: 1rem;
     cursor: pointer;
     font-size: 1.5rem;
+    transition: all 0.3s;
```

Comment:
Note that `all` is the default value for `transition-property`, so this is technically not necessary to specify it.